PodcastGenerator/themes/default/jumbotron.php

```diff
@@ -1,3 +1,4 @@
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
Should this be included here or somewhere better?

Reply:
We don't use CDNs, please download the file and put it somewhere in the core/ directory. Bootstrap is also somewhere there

Reply:
Opted for inline styling instead of adding a whole new CSS file. [commit b2b612a](https://github.com/PodcastGenerator/PodcastGenerator/pull/290/commits/b2b612a127810fad158dd7552ea91accec5b1bdd)